gabc/index.html

```diff
@@ -212,18 +202,15 @@ <h3 id="header">Header</h3>
 19th century revival), the name of the book; e.g. <code>Liber
 Usualis</code>.</dt>
 
+<dd><a name="language">language</a></dd>
+<dt>The language of the lyrics.</dt>
+
 <dd><a name="transcriber">transcriber</a></dd>
 <dt>The name of the transcriber into gabc.</dt>
 
 <dd><a name="transcription-date">transcription-date</a></dd>
 <dt>The date the gabc was written, with the following convention yyyymmdd, like 20090129 for january the 29th 2009.</dt>
 
-<dd><a name="gregoriotex-font">gregoriotex-font</a></dd>
-<dt>The font used
-by gregoriotex. At present gregoriotex can use three fonts:
-<code>greciliae</code>, <code>gregorio</code>, and
-<code>parmesan</code>.</dt>
-
 <dd><a name="mode">mode</a></dd>
 <dt>The mode of the piece.  This should
```

Comment:
`mode` *is* actually now used for the text above the initial, if an annotation is not provided.